ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py

```diff
@@ -8738,6 +8738,39 @@ def test_torch_tensor_square(
     )
 
 
+# square_
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="torch.tensor",
+    method_name="square_",
+    dtype_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
```

Comment:
The dtypes should be valid and the unsupprted dtypes should be added in the decorator above the function definition.